frontend/db.json

```diff
@@ -0,0 +1,65 @@
+{
+    "meals": [
+        {
+            "name": "Breakfast", 
+            "date_time_start": "1/22/22",
+            "date_time_end": "1/22/22",
```

Comment:
A couple of things here

- Can we call it "datetime_start" instead of "date_time_start" to match the API spec
- I forgot to specify the time format in the spec, its updated now. Can we use time in the format of "2022-01-24T23:46:25.872Z" ? You can easily get this by opening your browser console and typing `new Date().toISOString()`